operator/roles/kiali-remove/defaults/main.yml

```diff
@@ -1,6 +1,8 @@
 kiali_defaults:
   deployment:
     namespace: "istio-system"
+  auth:
```

Comment:
trivial - the top level "auth" should appear alphabetically above "deployment" so move this above like this:

```
auth:
  oauth_name: ""
deployment:
  namespace: "istio-system"
```